admin_user_reports.php

```diff
@@ -0,0 +1,232 @@
+        if(mysqli_query($db,$sql)){
+            $deactivate_result='You have sucessfully activated the user.';
+        }
+    }
+    if(isset($_POST['deactivateAccount']))
+    {
+        $accountID = $_POST['switchStatusID'];
+        echo 'Account ID: '.$accountID;
+        $sql = "UPDATE `1User` SET status = 'deactivated' WHERE  accountID='".$accountID ."'";
+        if(mysqli_query($db,$sql)){
+            $deactivate_result='You have sucessfully deactivated the user.';
+        }
+    }
+}
+?>
+<HTML>
+<HEAD>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.css">
+    <script src="functions.js">
+    </script>
+</HEAD>
+
+<BODY>
+<?php require 'admin_dashboard_navbar.php' //nav bar
+?>
+<H1>User Reports</H1>
+<table style="width: 100%;">
+    <tr>
+        <td style="text-align: center;" >
+            <table class="blueTable" style="margin-left: 3%;"">
+            <thead>
+            <tr>
+                <th>Account ID</th>
+                <th>Account Type</th>
+                <th>Account Category</th>
+                <th>Charge</th>
+                <th>Status</th>
+                <th>Email</th>
+                <th>Balance</th>
+                <th>Deactivate</th>
+            </tr>
+            </thead>
+            <tbody id="tableBody">
+            <?php
+            //Reset table to all applications
+            if(isset($_POST['resetTable']))
+            {
+                $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance FROM `1User` ORDER BY isEmployer DESC";
+                $result = mysqli_query($db,$sql);
+            }
+            // See Employers
+            elseif(isset($_POST['seeEmployers']))
+            {
+                $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance 
+                        FROM `1User`
+                        WHERE isEmployer='" . 1 ."'
+                        ORDER BY isEmployer DESC";
+                $result = mysqli_query($db,$sql);
+            }
+            //See JS
+            elseif(isset($_POST['seeJS']))
+            {
+                $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance 
+                        FROM `1User`
+                        WHERE isEmployer='" . 0 ."'
+                        ORDER BY isEmployer DESC";
+                $result = mysqli_query($db,$sql);
+            }
+            // Search by premium option
+            elseif(isset($_POST['filterPremium']))
+            {
+                $category = $_POST['accountCategory'];
+                if($category == 'all' || empty($category))
+                {
+                    $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance FROM oyc353_1.`1User`ORDER BY isEmployer DESC";
```

Comment:
maybe remove the _oyc353_1._ if it is not necessary

Reply:
Done